### ISSUE TITLE
imdb.py and download-imdb.py have the same contents ... ?

### ISSUE THREAD

**Issue by tedd4u** [lukas/ml-class#39]
https://github.com/lukas/ml-class/blob/master/videos/cnn-text/download-imdb.py
https://github.com/lukas/ml-class/blob/master/videos/cnn-text/imdb.py

I think the intended contents of download-imdb.py may have been replaced with the (correct) contents of imdb.py.

Thanks for the videos ... no idea why you are not getting more views, likes and & subscribes. Maybe try to do some ASMR or unboxing ;)

**Comment by jmetzz** [comment]
it should be this:
```bash
curl http://ai.stanford.edu/~amaas/data/sentiment/aclImdb_v1.tar.gz
tar xvfz aclImdb_v1.tar.gz
```

**Comment by charlesfrye** [comment]
Thanks for bringing this to our attention! This has been fixed in #64.
